#! /bin/sh /usr/share/dpatch/dpatch-run
## 50_debianize.dpatch by jre <jre-phoenix@users.sourceforge.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: - Change defaults and blockcontrol.conf to default to full
## DP:   Debian compatibility.

@DPATCH@
diff -urNad blockcontrol~/blockcontrol.conf blockcontrol/blockcontrol.conf
--- blockcontrol~/blockcontrol.conf	2009-01-16 19:21:27.000000000 +0100
+++ blockcontrol/blockcontrol.conf	2009-01-16 19:22:22.000000000 +0100
@@ -329,9 +329,9 @@
 # you can download one based on the Debian version from
 # moblock-deb.sourceforge.net.
 # 0 - Debian compatible system (uses start-stop-daemon instead of start_daemon)
-# 1 - LSB 3.1 compatible system
 #     (default)
+# 1 - LSB 3.1 compatible system
 # 2 - system with broken LSB 3.1 implementation, which doesn't understand the
 #     specification of the pidfile. Use this for Fedora Core 9 (and other redhat
 #     variants).
-LSB_MODE="1"
+LSB_MODE="0"
diff -urNad blockcontrol~/blockcontrol.conf.rej blockcontrol/blockcontrol.conf.rej
--- blockcontrol~/blockcontrol.conf.rej	1970-01-01 01:00:00.000000000 +0100
+++ blockcontrol/blockcontrol.conf.rej	2009-01-16 19:21:49.000000000 +0100
@@ -0,0 +1,19 @@
+***************
+*** 330,337 ****
+  # you can download one based on the Debian version from
+  # moblock-deb.sourceforge.net.
+- # 0 - Debian compatible system (uses start-stop-daemon instead of start_daemon)
+- # 1 - LSB 3.1 compatible system (default)
+  # 2 - system with broken LSB 3.1 implementation, which doesn't understand the
+  #     specification of the pidfile. Use this for Fedora Core 9 (and other redhat
+  #     variants).
+- LSB_MODE="1"
+--- 330,337 ----
+  # you can download one based on the Debian version from
+  # moblock-deb.sourceforge.net.
++ # 0 - Debian compatible system (uses start-stop-daemon instead of start_daemon) (default)
++ # 1 - LSB 3.1 compatible system
+  # 2 - system with broken LSB 3.1 implementation, which doesn't understand the
+  #     specification of the pidfile. Use this for Fedora Core 9 (and other redhat
+  #     variants).
++ LSB_MODE="0"
